aikau/src/main/resources/alfresco/css/less/defaults.less

```diff
@@ -1,3 +1,17 @@
+// Main palette
+@primaryBG: #fff;
+@primaryFG: #333;
+@headerBG: #0082c8;
```

Comment:
Theme-wise, it'd be good to have it as off-black, because it's a bit more stylish. I don't know why. Just off-black is the new black :)